app/components/EntityForm/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react';
+import { Control, Form } from 'react-redux-form/immutable';
+
+class EntityForm extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  render() {
+    return (
+      <Form
+        model={this.props.model}
+        onSubmit={this.props.handleSubmit}
+      >
+        {
+          this.props.fields.map((el, index) => (
+            <span key={index}>
+              <label htmlFor={el.id}>{el.label}</label>
+              { el.type === 'text' &&
+                <Control.text id={el.id} model={el.model} />
```

Comment:
I think it would be good to spread props here, 

```
const { id, label, model,...props } = this.field
```
then we can always pass anything extra down to the Control

<Control id={name} model={model || `.${name}`} {...props} >